packages/web-components/src/components/cbp-list/cbp-list.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * @prop --cbp-list-item-color: var(--cbp-color-text-darkest);
+ * @prop --cbp-list-item-color-dark: var(--cbp-color-text-lightest);
+*/
+
+:root {
+  --cbp-list-item-color: var(--cbp-color-text-darkest);
+  --cbp-list-item-color-dark: var(--cbp-color-text-lightest);
+  --cbp-list-padding: var(--cbp-space-4x);
+  --cbp-list-item-padding: 2px;
+  // --cbp-list-item-bullet: '-';
+}
+
+
+[data-cbp-theme=light] cbp-list[context*=dark]:not([context=light-always]),
+[data-cbp-theme=dark] cbp-list:not([context=dark-inverts]):not([context=light-always]) {
+  --cbp-list-item-color: var(--cbp-list-item-color-dark);
+}
+
+cbp-list {
+  line-height: var(--cbp-line-height-xs);
+
+  ol, ul{
+    padding-inline-start: var(--cbp-list-padding);
+  }
+  
+  li {
+    color: var(--cbp-list-item-color);
+    font-size: var(--cbp-font-size-body);
+    font-weight: var(--cbp-font-weight-bold);
+    padding-block: var(--cbp-list-item-padding);
+  }
+
+  
+  // //Unordered nested lists
+  ul > li{
```

Comment:
Do you need the >? I would think the nested ones would automatically have higher specificity.

Reply:
direct child there is so that when they are nested in the other we start at the first marker in type. I was having some cross pollination between ul and ol for sub lists